packages/@react-aria/tree/src/useTreeItem.ts

```diff
@@ -59,7 +59,7 @@ export function useTreeItem<T>(props: AriaTreeItemOptions, state: TreeState<T>,
         state.selectionManager.setFocusedKey(node.key);
       }
     },
-    tabIndex: isAndroid() ? -1 : null,
+    excludeFromTabOrder: true,
```

Comment:
Pretty sure this makes `tabIndex={-1}` all the time? I think it was intentional that there is no tabIndex other than on Android. @LFDanLu?

Reply:
Yeah, we'll have to test the above change on other screenreaders. Before I suggested this change I tried tracking down why exactly there was the `null` case but the blame lands here: https://github.com/adobe/react-spectrum/commit/9ade4865864e060a4ac33782c8ae03f5e2f6c4de which isn't very useful. It seemed to work fine on my test iPad VO so not sure if it was for some other reason